README.md

```diff
@@ -29,6 +29,18 @@ Pre-requisites:
 - NetBackup 8.1.1 or higher
 - See script README for perl requirements and usage
 
+#### Executing the snippets in node.js
+Pre-requisites:
+- NetBackup 8.1.1 or higher
+- node.js v8.9.4 or higher
+- node.js modules: `https, fs, stdio`
+
+Note: To install node.js module, Run `npm install <module_name>`.
+
+Use the following commands to run the node.js samples.
+- `node getHostsList.js --nbmaster <masterServer> -- port <portNumber> --username <username> --password <password> --domain <domainName>`
```

Comment:
I suggest these should be renamed to -nbmaster, username, password, domainName and domainType etc to be consistent with other scripts.

Reply:
I tried but with the third party library stdio I am unable to incorporate this. I also tried with other module as well but all are support -- option.